AUTHORS.md

```diff
@@ -0,0 +1,22 @@
+# Credits
+
+## Development Leads
+
+* **Cara Reisle** - *Team lead* - [GitHub](https://github.com/creisle)
+* **Morgan Bye** - *Voice of experience* - [morganbye.com](http://morganbye.com)
+
+## Original HackSeq-ers
+
+* **Yee Mey Seah** - [GitHub](https://github.com/yeemey)
```

Comment:
you can remove Matthew, and Yee Mey. They were not able to attend

Reply:
Removed